vignettes/introduction-to-partition.Rmd

```diff
@@ -137,14 +173,14 @@ There are several other partitioners, which all have names in the format `part_*
 To apply a different partitioner, use the `partitioner` argument in `partition()`.
 
 ```{r}
-prt_kmeans <- partition(df, threshold = .5, partitioner = part_kmeans())
-prt_kmeans
+prt_pc1 <- partition(baxter_otu, threshold = .5, partitioner = part_pc1())
+prt_pc1
 ```
 
 
 ## Permutation tests
 
-Data sets with variables that are independent tend to reduce only at lower thresholds because too much information is lost. In this set of 10 independent variables, setting the threshold to .5 results in a partition with no reduction. The original data is returned.  
+Data sets with variables that are independent tend to reduce only at lower thresholds because reductions lose too much information at higher thresholds. In this set of 10 independent variables, setting the threshold to 0.5 results in a partition with no reduction. `partition()` returns the original data.  
```

Comment:
Suggested edits:
Data sets with statistically independent variables reduce only at lower thresholds because variables can only be reduced if a substantial proportion of information is discarded. In this set of 10 independent variables, setting the threshold to 0.5 results in a partition with no reduction. `partition()` returns the original data.